README.md

```diff
@@ -7,15 +7,19 @@ A DAV client to send requests to DAV servers.
 Here's an example usage:
 
 ```javascript
-import { DAVClient, getInbox, HTTPFetchClient } from 'dav-client';
+import { DAVClient, getCalendarIcs, HTTPFetchClient } from 'dav-client';
 
 const [username, password] = ['admin@open-paas.org', 'secret'];
 const httpClient = new HTTPFetchClient();
 const davClient = new DAVClient({ baseURL: 'http://0.0.0.0:8001', httpClient, headers: { Authorization: `Basic ${Buffer.from([username, password].join(':')).toString('base64')}`} });
 
-getInbox(davClient)('5f60334e78d1b021351f9f6e')
+getCalendarIcs(davClient)('5f60334e78d1b021351f9f6e', 'inbox')
```

Comment:
I think we can let getInbox for now.

getCalendar is more generic and need more parameters in the XML REPORT to work.

Eventually getInbox may leverage a getCalendar.

Reply:
OK I'll roll back to `getInbox`.